rocAL/rocAL/source/rali_api_meta_data.cpp

```diff
@@ -216,9 +216,9 @@ RALI_API_CALL raliGetImageId(RaliContext p_context,  int* buf)
 }
 
 void
-RALI_API_CALL raliGetImageLabels(RaliContext p_context, int* buf)
+RALI_API_CALL raliGetImageLabels(RaliContext p_context, void* buf)
 {
-    
+
     if (!p_context)
```

Comment:
check if buf is valid (not null) and throw error if so. Applies to all the functions below